plugins/pullvals.py

```diff
@@ -59,12 +58,12 @@ def pullvals(histpair,
             # Bin 2 data
             bin2 = ref_hist.GetBinContent(x, y)
 
-            # Proper Poisson error
+            # TEMPERARY - Getting Symmetric Error - Need to update with >Proper Poisson error 
             if ref_hist.InheritsFrom('TProfile2D'):
                 bin1err = data_hist.GetBinError(x, y)
                 bin2err = ref_hist.GetBinError(x, y)
             else:
-                bin1err, bin2err = get_poisson_errors(bin1, bin2)
+                bin1err, bin2err = bin1**(.5), bin2**(.5)
```

Comment:
Is there a reason why we stopped using Poisson errors?

Reply:
The ROOT error is currently causing timeout issues. This is a **temporary fix**! We will fix this and have a supplementary PR fixing the asymmetric Poisson errors.

Reply:
Thanks! It's really important at this point. As long as there is enough entry in each bin, so we can approximate the error by Gaussian error, otherwise, Poisson is the correct way.